guides/common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc

```diff
@@ -64,7 +61,7 @@ server = _{foreman-example-com}_
 # chkconfig puppet on
 ----
 ifdef::satellite[]
-* On {EL} 7 and 8:
+* On {EL} 7, 8, and 9:
 endif::[]
 ifndef::satellite[]
 * On other operating systems:
```

Comment:
I can't reply inline, but line 74 states:

> Run the Puppet agent on the host:

But the command is `puppet ssl bootstrap`, which only ensures the certificates are set up. Running the agent is typically interpreted as also applying the catalog.

Reply:
And which is needed here? Both?

Reply:
Looking closer: the order is surprising. Normally you:

* Run `puppet ssl bootstrap` to request a certificate
* Sign the request
* Enable the service

If you start the service, it will also submit a certificate request, but you have less control over it. So running `puppet ssl bootstrap` after enabling the service is redundant.

Reply:
I'll remove it.